.env.example

```diff
@@ -1,3 +1,6 @@
+# This URL will work for Camunda SaaS environments; you will need to change it for other environments.
+ZEEBE_AUTHORIZATION_SERVER_URL=https://login.cloud.camunda.io/oauth/token
```

Comment:
@pepopowitz Can this be removed?

Reply:
I wouldn't remove it! To remind you of how this line got added: 

We were trying to run the code against the dev environment, which uses a different URL for authorization. The auth code supports overriding the default URL, which is the one in this file, but we didn't realize that until we looked at the auth code. 

By adding this in the .env.example, this would be a better cue to people that they can override this value if they are working in a non-production-SaaS environment.